libnavigation-metrics/src/test/java/com/mapbox/navigation/metrics/MapboxMetricsReporterTest.kt

```diff
@@ -34,25 +45,27 @@ class MapboxMetricsReporterTest {
     }
 
     @Test
-    fun telemetryPushCalledWhenAddValidEvent() {
+    fun telemetryPushCalledWhenAddValidEvent() = coroutineRule.runBlockingTest {
         val mapboxTelemetry = initMetricsReporterWithTelemetry()
         val metricEvent = StubNavigationEvent(NavigationMetrics.ARRIVE)
         val event = metricEvent.toTelemetryEvent()
 
         MapboxMetricsReporter.addEvent(metricEvent)
 
         verify { mapboxTelemetry.push(event) }
+        unmockkObject(ThreadController)
```

Comment:
Can this be done in `@After`?

Reply:
Let's be explicit and only add `mockkObject(ThreadController)` from `initMetricsReporterWithTelemetry` https://github.com/mapbox/mapbox-navigation-android/pull/2415/files#diff-04985ebb0b77c4ed5d8ba7df720949c3R90 (and keep `unmockkObject(ThreadController)`) for the tests needed. What do you think?